app/js/lib/foxbox/recipes.js

```diff
@@ -198,36 +199,53 @@ export default class Recipes {
     )
     .then(setters => {
       return setters.map(setter => {
-        let name, options = [];
+        let name;
+        let options = [];
 
         // Check for the complex extension kind
         if (typeof setter.kind === 'object') {
           if (setter.kind.kind === 'Sentence') {
             name = 'say';
-            options.push(...[{
-              label: '"Good morning!"',
-              value: { String: '"Good morning!"' }
-            }, {
-              label: '"Good afternoon!"',
-              value: { String: '"Good afternoon!"' }
-            }, {
-              label: '"Good evening!"',
-              value: { String: '"Good evening!"' }
-            }]);
+            options.push(...[
+              {
+                label: '"Good morning!"',
+                value: { String: '"Good morning!"' }
+              },
+              {
+                label: '"Good afternoon!"',
+                value: { String: '"Good afternoon!"' }
+              },
+              {
+                label: '"Good evening!"',
+                value: { String: '"Good evening!"' }
+              },
+            ]);
           }
         } else if (setter.kind === 'TakeSnapshot') {
-          name = 'Camera';
+          name = 'camera';
           options.push({
-              label: 'take a picture',
-              value: { 'Unit': null }
+            label: 'takes a picture',
```

Comment:
Because we use the labels to make phrases, I had to remove the upper case and make sure it's still proper English when concatenated.